contrib/inventory/stacki.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2016, Hugh Ma <hugh.ma@flextronics.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+# Stacki inventory script
+# Configure stacki.yml with proper auth information and place in the following:
+#  - ../inventory/stacki.yml
+#  - /etc/stacki/stacki.yml
+#  - /etc/ansible/stacki.yml
+# The stacki.yml file can contain entries for authentication information
+# regarding the Stacki front-end node.
+#
+# use_hostnames uses hostname rather than interface ip as connection
+#
+#
+
+"""
+Example Usage:
+    List Stacki Nodes
+    $ ./stack.py --list
+
+
+Example Configuration:
+---
+stacki:
+  auth:
+    stacki_user: admin
+    stacki_password: abc12345678910
+    stacki_endpoint: http://192.168.200.50/stack
+use_hostnames: false
+"""
+
+import argparse
+import os
+import sys
+import yaml
+from distutils.version import StrictVersion
+
+try:
+    import json
+except:
+    import simplejson as json
+
+try:
+    import requests
```

Comment:
actually, i just remembered i did implement strictversion below

`if StrictVersion(requests.__version__) < StrictVersion("2.4.3"):
     sys.exit('requests>=2.4.3 is required for this inventory script')`